components/ui/Button.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 const Button = ({ children, link = '#', target, size = 'normal' }) => {
   const buttonStyle = classNames('bg-white text-black -tracking-stretch flex w-fit px-8.75 lg:px-12 gap-6 group', {
     'py-4 text-xl sm:text-3xl leading-7.5 sm:leading-9.5 rounded-5xl': size === 'large',
-    'py-3 leading-9 text-lg sm:text-2xl rounded-4xl': size !== 'large',
+    'py-3 text-lg sm:text-2xl rounded-4xl leading-6.75 lg:leading-7.5': size !== 'large',
```

Comment:
well done @jacob-js,  but here is a few things. I should  request to change,
- on leading-7.5 instead of using lg , I think you can change to md 
-  letter spacing  it changes on small button of mobile breakpoint
here is how it should look
-not large (button)  on mobile -0.0125em
                             on desktop  -0.015em
-large button  on mobile  and desktop -0.015 em